recipes/vcftools/0.1.11/build.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
-
+set -euo pipefail
+sed -i.bak -e 's/DIRS = cpp perl/DIRS = cpp/' ./Makefile
```

Comment:
Prefixing a path with `./` is not necessary. Just write `Makefile` only. (Same for other file names below.)